### PR TITLE
docs: Update vimdoc file

### PR DESCRIPTION
**Pull request by vigoux**
Adds the new maintainers (that were actually added a long time ago), fix inconsistent formatting, and unneeded or old/wrong docs.

I think we could also start to include some sponsoring notes to the
repo, as suggested in https://github.com/nvim-treesitter/nvim-treesitter/pull/1142#issuecomment-819753698

### CODE REVIEW

**Review by kyazdani42 — APPROVED** [review]

File: doc/nvim-treesitter.txt [hunk 1]
```diff
@@ -175,71 +178,52 @@ Supported options:
...
-use is :
-- If you want to rewrite (or write) a query, don't use `after/queries`.
-- If you want to override a part of a query (only one match for example), use the `after/queries`
-  directory.
-
 ==============================================================================
 COMMANDS                                            *nvim-treesitter-commands*
 
-                                                                  *:TSInstall*
+								  *:TSInstall*
```
Comment (stsewd) [inline doc/nvim-treesitter.txt:184]: I'm more fan of spaces, but all good p:

**Review by stsewd — APPROVED** [review]

**Comment by stsewd** [comment]
> I think we could also start to include some sponsoring notes to the
repo, as suggested in #1142 (comment)

I think you can apply to https://github.com/sponsors/accounts there you can choose the organization (they don't charge any fees from what I understand), but I'm not familiar how it works, and I think it isn't available for all countries

**Comment by vigoux** [comment]
To apply to organization sponsoring we'll eed to have a financial host set up, something like open collectives (that's how the neovim organization do).
I'll see how I can set that up tomorrow.

File: doc/nvim-treesitter.txt (hunk 1 above)
Comment (vigoux) [reply doc/nvim-treesitter.txt:184]: Default help style is with tabs, so we go for tabs.

**Review by theHamsta — APPROVED** [review]

**Pull request merged**